src/services/riotApiClient.test.js

```diff
@@ -136,11 +137,16 @@ describe('Fetching the latest patch version', () => {
   })
 
   test('Riot API Client throws an error when the GET request fails.', async () => {
+    let result = {}
+    result.data =jest.fn()
+    logger.error = jest.fn()
```

Comment:
There may be another way. Look up the docs on the logger library we are using to see if you can turn off logging instead of mocking the error function.